src/Module.cpp

```diff
@@ -752,15 +752,18 @@ class ScopedCompilerLogger {
 void compile_multitarget(const std::string &fn_name,
                          const std::map<Output, std::string> &output_files,
                          const std::vector<Target> &targets,
+                         const std::vector<std::string> &suffixes,
                          const ModuleFactory &module_factory,
                          const CompilerLoggerFactory &compiler_logger_factory) {
     validate_outputs(output_files);
 
     user_assert(!fn_name.empty()) << "Function name must be specified.\n";
     user_assert(!targets.empty()) << "Must specify at least one target.\n";
+    user_assert(suffixes.empty() || suffixes.size() == targets.size())
+        << "The suffixes list must be empty or the same length as the targets list.\n";
 
     // The final target in the list is considered "baseline", and is used
-    // for (e.g.) the runtime and shared code. It is often just os-arch-bits
+    // for (e.g.) the runtime and shared code. It is often just os-bits-arch
```

Comment:
linux-64-x86? I thought we usually did linux-x86-64

Reply:
Canonical form generated by `Target::to_string()` is `arch-bits-os` so both of the above are wrong, will fix